data.sql

```diff
@@ -31,5 +31,6 @@ VALUES ('Boarmon', '06-07-2005', 7, true, 20.4),
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
 VALUES ('Boarmon', '06-07-2005', 7, true, 20.4), 
 
+
```

Comment:
* Kindly Insert the following data into the owners table:

> Sam Smith 34 years old.
> Jennifer Orwell 19 years old.
> Bob 45 years old.
> Melody Pond 77 years old.
> Dean Winchester 14 years old.
> Jodie Whittaker 38 years old
* And also [these requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following,data.sql%20file.) are missing kindly insert all the necessary datas